### ISSUE TITLE
average_precision_at_k is imported differently in TF 2.0

### ISSUE THREAD

**Issue by Erlemar** [LucaCappelletti94/extra_keras_metrics#4]
Hello, thank you for your library.
I have noticed that in TF 2.0 the import of average_precision_at_k has changed:
Previously it was 
`tf.metrics.average_precision_at_k`
Now it is:
`tf.compat.v1.metrics.average_precision_at_k`

*UPD*:
also `tf.to_int64` changed to `tf.compat.v1.to_int64` and `get_session` is not available when using TensorFlow 2.0.

**Comment by LucaCappelletti94** [comment]
Yes, I am aware I still have to fully update the library for Tensorflow 2. Currently, I'm in the middle of numerous research projects, so I will not be able to tackle these issues soon.

I will put a warning in the README about this.

**Comment by LucaCappelletti94** [comment]
Should be working fine now
